src/components/WorldMap/index.js

```diff
@@ -139,18 +141,39 @@ class WorldMap extends Component {
         return crimeA.distanceToUser - crimeB.distanceToUser;
       });
 
-      warCrimes[0].classList.add('closest');
+      warCrimes.forEach(crime => {
+        crime.classList.add('closest');
+        crime.style.animationPlayState = 'paused';
+      });
       warCrimes[0].style.animationPlayState = 'running';
+
+      this.props.setClosestCase(warCrimes[0].toString());
+
+      this.setState({ closestWarcrime: warCrimes[0].id });
+      this.setState({ allWarCrimes: warCrimes });
```

Comment:
kun je hier niet ook de ref voor gebruiken, net als aan het begin van `calculateDistanceToUser`? state setten triggered een rerender wat hier misschien niet nodig is aangezien we echt alleen DOM operaties doen en geen applicatie state veranderen